src/App.jsx

```diff
@@ -22,6 +23,14 @@ const router = createBrowserRouter([
             </GuestProtectedRoute>
         ),
     },
+    {
+        path: 'signup',
+        element: (
+            <GuestProtectedRoute>
```

Comment:
why signup is a Protected Route ?

Reply:
Component GuestProtectedRoute hides the signup page from logged in users.

Reply:
do we have a logout button? how can i login as a different user

Reply:
We don't. Should i add it?

Reply:
done